state/state-network.md

```diff
@@ -65,7 +65,18 @@ custom_payload = SSZ.serialize(custom_data)
 
 #### POKE Mechanism
 
-The [POKE Mechanism](./portal-wire-protocol#poke-mechanism) MUST be disabled for the state network. As `content_for_retrieval` is different from `content_for_offer` the POKE mechanism cannot offer content that is verifiable.
+As `content_for_retrieval` is different from `content_for_offer` the POKE mechanism cannot offer content that is verifiable without providing
+the proof/s that are contained in the `content_for_offer` payload. These proofs are usually available when walking down the trie during content
+lookups such as during an `eth_getBalance` JSON-RPC call implemented in the state network.
+
+The [POKE Mechanism](./portal-wire-protocol#poke-mechanism) for the state network requires building a `content_for_offer` by combining the `content_for_retrieval` with the parent proof/s and block hash. This is implemented differently for each
+type of content:
+- For account trie nodes the trie node in the `content_for_retrieval` is appended to the parent account proof and then combined with the block hash to build the `content_for_offer`.
+- For contract trie nodes the trie node in the `content_for_retrieval` is appended to the parent storage proof and then combined with the account proof and block hash to build the `content_for_offer`.
+- For contract code the code in the `content_for_retrieval` is combined with the account proof and block hash to build the `content_for_offer`.
+
+This POKE mechanism as described above SHOULD be executed after looking up content from the network, whenever the proofs and block hash are available
+to be used to build the `content_for_offer`.
```

Comment:
```suggestion
This POKE mechanism as described above SHOULD be executed after looking up content from the network, whenever the proofs and block hash are locally available
to be used to build the `content_for_offer`.
```

Small suggestion to avoid any implementation doing additional network requests for this (although it would not be needed in this lookup scenario), as that would be more costly (and more abusable).